### DISCUSSION TITLE
Middle-click of wheel mouse

### DISCUSSION THREAD

**Discussion by pamoroso** [joncampbell123/dosbox-x#5141] [Q&A] [answered]
How can I check whether a wheel mouse is configured for 3-button operation? 

I'd like a click of the mouse wheel to generate a middle-click for software running under DOSBox-X. In the SDL section of the  configuration tool the option mouse_wheel_key is set to -1: is this the proper value for 3-button operation?

Under DOSBox-X on Linux Mint I'm using Medley Interlisp, the DOS port of a 1990s Lisp development environment that runs in VGA/SVGA. While the left and right mouse buttons work as expected, pressing the mouse wheel has no effect even when Medley Interlisp is supposed to carry out the corresponding action (e.g. showing a pop up menu).

**Comment by maron2000** [comment, marked as answer]
Wiki mentions about the 3rd button, if you haven't tried.

https://github.com/joncampbell123/dosbox-x/wiki/Guide%3AMouse-support-in-DOSBox%E2%80%90X#auxdevice

**Comment by pamoroso** [comment]
Thank you very much, that did the trick. I set the following configuration options:

* Main > Configuration tool > Keyboard > auxdevice > 3button
* Main > Configuration tool > SDL > middle_unlock > none

and now Medley Interlisp for MS-DOS recognizes the middle mouse button.
